pkg/steps/release/promote.go

```diff
@@ -248,6 +259,14 @@ func getPromotionPod(imageMirrorTarget map[string]string, namespace string) *cor
 	}
 }
 
+func tagInQuay(image string, date string) (string, error) {
```

Comment:
Can we have a unittest for this function?

Reply:
Done.